mgmtsystem/ec2.py

```diff
@@ -537,3 +540,50 @@ def delete_loadbalancer(self, loadbalancer):
 
     def get_all_unused_network_interfaces(self):
         return [eni for eni in self.api.get_all_network_interfaces() if eni.status == "available"]
+
+    def import_image(self, s3bucket, s3key, format="vhd", description=None):
+        self.logger.info(" Importing image {} from {} bucket with description {} in {} started "
+                         "successfully."
+                         "format".format(s3key, s3bucket, description, format))
+        try:
+            result = self.ec2_connection.import_image(DiskContainers=[
+                {
+                    'Description': description if description is not None else s3key,
+                    'Format': format,
+                    'UserBucket': {
+                        'S3Bucket': s3bucket,
+                        'S3Key': s3key
+                    }
+                }
+            ])
+            image_id = result.get("ImageID")
+            return image_id
```

Comment:
yeah I'd agree with your comment in the PR description - return tuple with task ID